### PR TITLE
FIX required body in user/follow post

### PR DESCRIPTION
**Pull request by mapy69003**
API crashes because of an empty body.

### CODE REVIEW

File: routes/user.js [hunk 1]
```diff
@@ -80,6 +80,14 @@ exports.findByName = function (req, res) {
 };
 
 exports.follow = function (req, res) {
+    console.log(req.body.id);
+    if (typeof req.body == 'undefined' || typeof req.body.id == 'undefined') {
```
Comment (vseguin) [inline routes/user.js]: Please use triple equality here.

File: routes/user.js [hunk 2]
```diff
@@ -80,6 +80,14 @@ exports.findByName = function (req, res) {
...
+            res.status(412).send({
```
Comment (vseguin) [inline routes/user.js]: Please never send a 412. Use a 400.

File: routes/user.js [hunk 3]
```diff
@@ -80,6 +80,14 @@ exports.findByName = function (req, res) {
 };
 
 exports.follow = function (req, res) {
+    console.log(req.body.id);
```
Comment (vseguin) [inline routes/user.js]: Remove this console.log

**Comment by mapy69003** [comment]
Done

**Pull request merged**